src/components/AppComponent.js

```diff
@@ -66,7 +66,7 @@ export const AppComponent = connect(({ dataNonce, focus, search, showContexts, u
       }
     }
   }
-
+  const isLoading = store.getState().isLoading
```

Comment:
Please pass `isLoading` through the Redux `connect` function rather than pulling directly from state.